pycqed/measurement/detector_functions.py

```diff
@@ -781,6 +824,127 @@ def prepare(self, sweep_points):
                                           mode='iavg')
 
 
+class UHFQC_scope_detector(Hard_Detector):
```

Comment:
I haven't checked all details of this class since it is a new class and cannot break anything else in pycqed.

The class contains a lot of UHF-specific code, and we will have to check in the future how to make it work for other acquisition devices (SHFQA, FPGA and possibly oscilloscope, VNA)